### DISCUSSION TITLE
Ability to add robots.txt

### DISCUSSION THREAD

**Discussion by octocolby** [datalust/seq-tickets#1341] [Help]
Hi,

Is there a way to add a robots.txt file or set the X-Robots-Tag http header without setting the app behind a reverse proxy?
I haven't been able to see anything in the configuration. Any help would be much appreciated.

**Comment by nblumhardt** [comment]
Hi! No, there's nothing like this, today. My guess is that you want robots.txt to instruct robots not to crawl/index Seq? This might make a nice general enhancement..... 🤔

**Reply by octocolby** [reply]
Yep. Exactly what I am after 😄 . I can always use a reverse proxy if I really need but having a robots.txt as a default makes sense to me for this type of application.

**Reply by nblumhardt** [reply]
Awesome - thanks! Posted a new issue and mentioned you - we should be able to squeeze this into 2021.3.
